.github/workflows/test-spec.yml

```diff
@@ -28,45 +28,47 @@ jobs:
             admin_password: admin
           - version: 2.0.0
             admin_password: admin
-          - version: 2.19.1
-          - version: 2.19.1
+          - version: 3.0.0-beta1
+          - version: 3.0.0-beta1
             tests: plugins/index_state_management
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/ml
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: routing
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: snapshot
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: remote_store
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: dangling
             url: http://localhost:9200
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/replication
             url: http://localhost:9200
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/streaming
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/notifications
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/query_insights
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/workload-management
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/analysis
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/knn
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/learning_to_rank
-          - version: 2.19.1
+          - version: 3.0.0-beta1
             tests: plugins/security
             cert: tests/plugins/security/.kirk.pem
             key: tests/plugins/security/.kirk-key.pem
+          - version: 3.0.0-beta1
+            tests: plugins/neural
```

Comment:
There's already tests/plugins/ml that sets up ml-commons, do you need a separate run? Add to those tests.